### DISCUSSION TITLE
Append data to different sheet in one workbook?

### DISCUSSION THREAD

**Discussion by mtengmo** [dfinke/ImportExcel#1497] [Q&A]
I got a premade template with 1-10 rows in the beginning with headers and information on different sheet, I want to append data, is it possible to assign start row or just the last one without data?

**Comment by dfinke** [comment]
Something like this:

```powershell
$data | Export-Excel ./sales.xlsx -WorksheetName Jan -StartRow 4
```

Not sure what you mean by ` or just the last one without data`

**Comment by HLaksmana** [comment]
You are looking for the -Append option for export-excel. StartRow should be where your headers are

**Reply by HLaksmana** [reply]
Note, it will only export data for headers that match your destination
